lighthouse-cli/test/fixtures/byte-efficiency/tester.html

```diff
@@ -21,15 +21,15 @@
     if (used) {
       const div = document.createElement('div');
       div.classList.add(className);
-      document.body.appendChild(div);
+      document.body.append(div);
     }
     sizeInBytes -= rule.length * 0.2; // 1 byte per character, GZip estimate is 20% for Stylesheets
   }
 
   const style = document.createElement('style');
   const textContent = firstContent + data.join('\n');
-  style.appendChild(document.createTextNode(textContent));
-  document.head.appendChild(style);
+  style.append(document.createTextNode(textContent));
```

Comment:
No need to wrap with document.createTextNode, append does that for strings already.